src/components/Modal/Modal.vue

```diff
@@ -498,15 +498,23 @@ $header-size: 50px;
 	.modal-title {
 		overflow-x: hidden;
 		box-sizing: border-box;
-		max-width: 100%;
-		padding: 0 #{$clickable-area * 2}; // maximum actions is 2
+		width: 100%;
+		padding: 0 #{$clickable-area * 3} 0 12px; // maximum actions is 2 but we violate it here
 		transition: padding ease 100ms;
 		white-space: nowrap;
 		text-overflow: ellipsis;
 		color: #fff;
 		font-size: $icon-margin;
 	}
 
+	// On wider screens the title can be centered
+	@media only screen and (min-width: $breakpoint-mobile/2) {
```

Comment:
half the mobile breakpoint?

Reply:
Yes, cause it looks bad to already do it at the mobile breakpoint. Test and you will see. :)

Reply:
I understand, I wanted to make sure this was not a mistake :+1: